capabilityreceiver/pkg/capabilityhandler/capabilityreceiver_integration_test.go

```diff
@@ -132,9 +132,7 @@ func Test_saveMsgInDB(t *testing.T) {
 	th.Assert(t, err == nil, err)
 	th.Assert(t, ct == 1, "did not store data as expected")
 
-	// @TODO - Validation - This will need to be updated
 	storedEndpt, err := store.GetFHIREndpointInfoUsingURL(ctx, testFhirEndpoint1.URL)
-	storedEndpt.Validation = testValidationObj
 	th.Assert(t, err == nil, err)
 	th.Assert(t, expectedEndpt.Equal(storedEndpt), "stored data does not equal expected store data")
```

Comment:
Not sure if I am missing something, but it I believe you might need to include a validation ID for the expected endpoint to equal the stored endpoint?

Reply:
you're totally right, this is what i get for trying to rush to finish this before vacation!